_data/projects/PySwarms.yml

```diff
@@ -0,0 +1,11 @@
+name: PySwarms
+desc: A Particle Swarm Optimization (PSO) library in Python
+site: https://github.com/ljvmiranda921/pyswarms
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- python
+- optimization
+- algorithms
+upforgrabs:
+  name: Help wanted!
```

Comment:
This name should match up with the label you are assigning to issues, e.g. `help wanted`